openedx/core/djangoapps/discussions/docs/decisions/0001-discussion-configuration-api.rst

```diff
@@ -0,0 +1,102 @@
+Discussion provider configuration
+=================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+As part of the BD-03 initiative (Blended Development, Project 3),
+we want to enable third-party discussion providers to replace the
+default forums experience in edx-platform.
+
+To accomplish this, we should establish a configuration system to
+enable/disable/configure these new discussion-provider plugins;
+the existing/default/legacy forums experience will be included as one of these plugins.
```

Comment:
Nit: I dislike having strings of a/b/c when only talking about one thing. I'd think simply `existing forums experience` or `edX forums` would suffice